content/technotes/2020-02-26-tic-ghactions.Rmd

```diff
@@ -0,0 +1,176 @@
+---
+slug: "tic-ghactions"
+title: Supercharge your GitHub Actions Experience with tic
+# Delete the package_version line below if your post is not about a package
+package_version: 0.5.0
+authors:
+  - Patrick Schratz
+# Set the date below to the publication date of your post
+date: 2020-02-26
+# Set categories to technotes if this is a tech note
+categories: technotes
+# Leave topicid blank below; will be set by editor
+topicid:
+# Minimal tags for a post about a community-contributed package 
+# that has passed software peer review are listed below
+# Consult the Technical Guidelines for information on choosing other tags
+tags:
+  - R
+  - GitHub
+  - GitHub Actions
+  - Continuous Integration
+  - software-peer-review
+  - tic
+  - community
+# The summary below will be used by e.g. Twitter cards
+description: "tic aims to simplify Continuous Integration (CI) related tasks for R projects (not just packages by the way) and now provides supercharged support for GitHub Actions. By enhancing the good work of [r-lib/actions](https://github.com/r-lib/actions) tic provides a DSL for R for CI tasks."
```

Comment:
After checking the Twitter Card validator, I think we need to shorten this a bit, to make sure the important part (support for GitHub actions) is visible.
```suggestion
description: "tic simplifies CI tasks for R projects and now provides supercharged support for GitHub Actions. By enhancing the good work of [r-lib/actions](https://github.com/r-lib/actions) tic provides a DSL for R for CI tasks."
```